### ISSUE TITLE
Make titlebar light variant again

### ISSUE THREAD

**Issue by Corsseir** [flathub/com.spotify.Client#50]
It looks really ugly with dark variant, colors does not match and contrast between titlebar and rest of window is too low. Can you revert this change, please?

**Comment by Corsseir** [comment]
It would be nice. Thank you.

**Comment by jurf** [comment]
Spotify uses a _dark_ theme that **cannot be changed**. This is not a matter of taste, but consistency.

Let me try this way. This is Spotify under Windows:

![obrazok](https://user-images.githubusercontent.com/7472425/48148917-2efe9c80-e2bb-11e8-81cd-37fec5ae1bb7.png)

This is Spotify under macOS:

![obrazok](https://user-images.githubusercontent.com/7472425/48148827-fc54a400-e2ba-11e8-97c5-adc53bf9e668.png)

Even Spotify on Android uses both a dark top bar and application bar in the app switcher.

The dark variant on GNOME is there for this use-case – so that an app with a dark theme does not have a jarring white bar. This is especially bad when the app is maximised under GNOME:

![obrazok](https://user-images.githubusercontent.com/7472425/48149263-f6ab8e00-e2bb-11e8-9744-d67ce5e5c024.png)

Compare that to:

![obrazok](https://user-images.githubusercontent.com/7472425/48149295-03c87d00-e2bc-11e8-9ecc-b53097ead6d7.png)

You can also look at e.g. Blender, which is another non-GTK+ app with a dark theme, where this happens by default now (finally).

TL;DR: This is a bugfix, not a preference.

**Comment by Erick555** [comment]
My point is that this "bugfix" should be a local thing, not part of flatpak. If you install spotify from different sources there aren't such hacks included. Why flatpak should be different? I'm sure if spotify would release official flatpak they won't include such things.

**Comment by Corsseir** [comment]
Firstable, Spotify uses custom themed window borders on the other oses. In this case, background of window border is consistent to the whole app design. They even put controls on window border space. GNOME doesn't offer this sort of functionality to the developers. What this flatpak do is just switching window mode to the dark variant of GNOME theme that have nothing close to consistancy with Spotify app design. Look closer - background color of the top bar is not consistent with Spotify color palette. Besides, this hacky thing makes the window top bar flickering from light variant to dark variant on the beginning when app is opening. This is not the best user experience. If Spotify devs decided to not provide custom themed window border on Linux then app should uses the default variant by default.

**Comment by Corsseir** [comment]
It doesn't matter. The app - especially proprietary - should be provided in
shape their creators wished.

What this flatpak do is not workaround, it's just a dirty, low quality
(flickering) hack.

Anyway, I stopped using this flatpak some time ago because this whole "work
around. I moved back to classic solution - negativo spotify repo.

**Comment by TingPing** [comment]
I won't disagree that this is a hack. I will disagree that any creator or designer at Spotify wants it to look like this. There just clearly isn't any maintainer for the Linux version to make it use CSD.

**Comment by Corsseir** [comment]
If so, why then snap official package provides by Spotify doesn't have
enabled dark theme by default?

**Comment by outof-step** [comment]
Ubuntu already uses a dark theme by default [1]. This package is not mantained by Spotify AB because Spotify does not commit to support Linux distributions other than Ubuntu LTS. 

You are already running a hack by installing this Flatpak, because it wasn't packaged by the original developers. 

"""
Spotify for Linux is a labor of love from our engineers that wanted to listen to Spotify on their Linux development machines. They work on it in their spare time and it is currently not a platform that we actively support. [2]
""" 

[1]: https://snapcraft.io/spotify
[2]: https://www.spotify.com/cl/download/linux/

**Comment by Erick555** [comment]
> You are already running a hack by installing this Flatpak, because it wasn't packaged by the original developers.

Flatpak is based on official spotify deb package. The "hack" isn't an integral part of it. I'm not sure if we're talking about the same thing as in snapcraft screenshots.

**Comment by TingPing** [comment]
I added a method to override this:

`flatpak override --user --env=SPOTIFYFLATPAK_DISABLE_DARK_TITLEBAR=1 com.spotify.Client`
